stubs/flake8-bugbear/bugbear.pyi

```diff
@@ -0,0 +1,26 @@
+import argparse
+import ast
+from typing import Any
+
+class BugBearChecker:
+    name: str
+    version: str
+    tree: ast.AST | None
+    filename: str | None
+    lines: list[str] | None
```

Comment:
Consider using `Sequence` instead of `list`, as it is less restrictive and also works. Arbitrary iterables won't work, because the code does checks like `if blahblah or not self.lines`.